### PR TITLE
Securities table

### PR DESCRIPTION
**Pull request by lucaimbrogno**
holdings_table
First iteration of holdings table. also fixed securities value

### CODE REVIEW

File: packages/dashboard/src/views/Holdings.js [hunk 1]
```diff
@@ -190,6 +193,53 @@ const Holdings = ({
   // todo move this to only update on account change
   const accountSmallStats = renderSmallStats()
 
+  const renderTableData = () => {
+    // check if we have account  data
+    if (historicQuotes['1d'] && historicQuotes['all']) {
```
Comment (alexanderMontague) [inline packages/dashboard/src/views/Holdings.js:198]: what happens if the user is logged out or we dont have data? Is it a blank card/table or nothing

**Review by alexanderMontague — APPROVED** [review]
Nice this guys gettin the hang of this

File: packages/dashboard/src/views/Holdings.js [hunk 2]
```diff
@@ -190,6 +193,53 @@ const Holdings = ({
...
+                position.book_value.amount
+              ).toFixed(2)}
+          </td>
+          <td>
+            {(
+              ((position.quote.amount * position.quantity -
+                position.book_value.amount) /
+                position.book_value.amount) *
+              100
+            ).toFixed(2) + '%'}{' '}
```
Comment (alexanderMontague) [inline packages/dashboard/src/views/Holdings.js:228]: a good way of doing string interpolation is using template literals https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
just lets you sub variables and strings in easily ``const string = `value is: ${value}CAD` ``
just for future this is usually easier

File: packages/dashboard/src/views/Holdings.js (hunk 1 above)
Comment (lucaimbrogno) [reply packages/dashboard/src/views/Holdings.js:198]: just a blank table

File: packages/dashboard/src/views/Holdings.js (hunk 2 above)
Comment (lucaimbrogno) [reply packages/dashboard/src/views/Holdings.js:228]: Roger that

**Pull request merged**